src/app/workflow/versions/versions.component.ts

```diff
@@ -89,6 +89,7 @@ export class VersionsWorkflowComponent extends Versions implements OnInit, OnCha
   workflow: ExtendedWorkflow;
   entryType = EntryType;
   DoiInitiatorEnum = Doi.InitiatorEnum;
+  engineVersionsList: string[] = ['Nextflow !>=22.10.1', 'Nextflow !>=23.04.0'];
```

Comment:
A little curious about this hard coding, this doesn't look like a test at first glance

Reply:
Oops, my mistake, used this for local testing but forgot to remove it. Removed now :slightly_smiling_face: